### DISCUSSION TITLE
SEO/Google Search

### DISCUSSION THREAD

**Discussion by grimsteel** [For-0/vocabustudy#162] [General]
Does anyone have any ideas about improving the Google Search thing for Vocabustudy?
It saved my reputation when ms cuevas googled "vocabustudy"
![image](https://user-images.githubusercontent.com/115331356/197293922-531a4632-fd53-4a11-a2c2-311935a589cf.png)

**Comment by IMGROOT2** [comment]
maybe we can look online for some SEO tips and stuff, but really big companies like amazon have entire departments whose only job is to uncover the google search algorithm. honestly I have no clue lol

**Reply by grimsteel** [reply]
> honestly I have no clue lol

Same...

**Reply by IMGROOT2** [reply]
https://www.wordstream.com/blog/ws/2021/03/05/seo-strategy

hmmm

**Reply by IMGROOT2** [reply]
ok so you need vocab keywords in the front page of your website, so that when google searches your website for keywords, it shows up on the search engine as something that might be helpful.

**Reply by IMGROOT2** [reply]
https://tools.wordstream.com/fkt?website=vocab&cid=&camplink=&campname=

**Reply by grimsteel** [reply]
![image](https://user-images.githubusercontent.com/115331356/197296311-eab42216-0bda-41b5-9db4-7b0a57b01b60.png)

**Reply by IMGROOT2** [reply]
Yeah, you pretty much put all the keywords in it! maybe the front page name will have to change, like "Vocabustudy - Study and build Vocabulary for free"

**Reply by grimsteel** [reply]
Yes I will change it
